server/helpers/token.ts

```diff
@@ -0,0 +1,15 @@
+import jwt from 'jsonwebtoken';
+import { JWT_SECRET_KEY } from '../src/config';
+
+/** return signed JWT from user data. */
+
+function createToken(user: string) {
+
+    const payload = {
+        username: user
+    };
+
+    return jwt.sign(payload, JWT_SECRET_KEY);
+}
+
+export default createToken;
```

Comment:
```suggestion
export function createToken(user: string) {

    const payload = {
        username: user
    };

    return jwt.sign(payload, JWT_SECRET_KEY);
}
```

you can just export the function directly instead of `export default`

Reply:
you'll need to update your calling functions to `import { createToken }` though (which is clearer IMO)